barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_format.cpp

```diff
@@ -270,9 +273,8 @@ void build_constraints(Builder& builder, AcirProgram& program, const ProgramMeta
             // final recursion output.
             builder.add_pairing_point_accumulator(current_aggregation_object);
         }
-        // TODO(https://github.com/AztecProtocol/barretenberg/issues/1183): This assertion should be true, except for
```

Comment:
honk_recursion is 2 for the tube, base, merge, block root, block merge circuits, but NOT the root rollup.

This assert previously wasn't true because the root rollup wasn't running the full verifier and therefore still spit out an ipa claim and proof, but now that it does run full verification, it will output nothing.